src/lib/marlin_plonk_bindings/js/README.md

```diff
@@ -0,0 +1,20 @@
+This library provides a wrapper around the WebAssembly prover code, which
+allows `js_of_ocaml` to compile the mina project against the WebAssembly
+backend.
+
+The different versions of the backend are generated in subdirectories; e.g. the
+NodeJS backend is generated in `node_js/` and the Chrome backend is generated
+in `chrome/`. To use a backend, run `dune build backend/plonk_wasm.js` and copy
+`backend/plonk_wasm*` to the project directory.
+
+Note that the backend code is not automatically compiled while linking against
```

Comment:
Is there some way in dune to get the relevant files to be built if you depend on the library?

Reply:
I haven't been able to figure out how to do this yet. We can't list the javascript files for `js_of_ocaml` because it will try (and fail) to parse them for inlining and unused code elimination.

For the client SDK, we use a separate `Makefile` to create the complete package; it'll probably be easier to do similar here rather than trying to trick dune into building it.